htdocs/core/lib/functions.lib.php

```diff
@@ -588,7 +588,7 @@ function GETPOST($paramname, $check = 'alphanohtml', $method = 0, $filter = null
 				}
 			}
 			if (!empty($conf->global->MAIN_ENABLE_DEFAULT_VALUES)) {
-				if (!empty($_GET['action']) && (preg_match('/^create/', $_GET['action']) || preg_match('/^presend/', $_GET['action'])) && !isset($_GET[$paramname]) && !isset($_POST[$paramname])) {
+				if (!empty($_GET['action']) && (preg_match('/^create|^add|^make/', $_GET['action']) || preg_match('/^presend/', $_GET['action'])) && !isset($_GET[$paramname]) && !isset($_POST[$paramname])) {
```

Comment:
Adding the ^add will bring bad side effect because 'add( is action after posting a form shown with create. And default values must be triggered only on the create page. After a post of add, the value to keep is the value entered by user on form. May be we can add ^add_price
Adding ^make seems ok.

However, even if renaming action to have consistency may have effect, for a long term goal i think we should take the risk to do this into "develop" branch. Moving to consistency is a long road action and this is a new step we should dare into develop...

Reply:
Very good point about the impact on the 'add' action - I missed that.

I now changed it to add_price as a short term solution for v17 and will create another patch for develop to change to create_price instead.